settings.gradle

```diff
@@ -51,4 +51,10 @@ include 'eventuate-tram-spring-messaging-starter'
 include 'eventuate-tram-spring-events-publisher-starter'
 include 'eventuate-tram-spring-events-subscriber-starter'
 include 'eventuate-tram-spring-events-starter'
-include 'eventuate-tram-spring-commands-starter'
\ No newline at end of file
+include 'eventuate-tram-spring-commands-starter'
```

Comment:
Why add all of this? Also there are no module files. Adding KafkaConsumerConfigurerConfiguration.class to TramJdbcKafkaConfiguration for spring and having KafkaConsumerConfigurerFactory in classpaths for micronaut should be sufficient.